### DISCUSSION TITLE
No Console Only option in the latest UTM 4.3.5 - Blank Screen during Ubuntu installation

### DISCUSSION THREAD

**Discussion by pmessri** [utmapp/UTM#5554] [General]
I'm trying to install latest Ubuntu 22.04 ARM64 image as an emulator, but after I select to install I get a blank screen.  I saw somewhere that you can do a console only option (for the Kali Linux) but I don't have that option available.  I am getting a blank screen and I think that is the only way to fix it.  Please advise what I am missing.  Thanks.

**Comment by wh173d3v11** [comment]
Hey, for now you can create a new serial devices. It open the terminal, and kali blank screen issue will be solved.
<img width="621" alt="image" src="https://github.com/utmapp/UTM/assets/46423041/36fc415c-c414-4122-a28b-dffee728fb46">
